github/PullRequest.py

```diff
@@ -794,6 +794,36 @@ def merge(self, commit_message=github.GithubObject.NotSet, commit_title=github.G
         )
         return github.PullRequestMergeStatus.PullRequestMergeStatus(self._requester, headers, data, completed=True)
 
+    def add_to_assignees(self, *assignees):
+        """
+        :calls: `POST /repos/:owner/:repo/issues/:number/assignees <https://developer.github.com/v3/issues/assignees>`_
+        :param assignee: :class:`github.NamedUser.NamedUser` or string
```

Comment:
This is not the parameter. You are expecting a list and even iterating over it, please correct the docstring.

Reply:
@s-t-e-v-e-n-k corrected now. Was a copy-paste mistake, will pay more attention next time.